blog/_posts/2016-10-15-team-kindr3d-saying-bye.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Team kindr3d – Saying Bye!
+layout: post
+created_at: Tue Oct 15 2016
+permalink: blog/2016-07-19-team-kindr3d-saying-hi
```

Comment:
please make the permalink consistent with the date and title of your post (seems like it's the same permalink from your first post) :)